addons/skin.estuary/1080i/DialogPVRInfo.xml

```diff
@@ -90,11 +90,9 @@
 					<param name="visible" value="Window.IsActive(PVRGuideInfo)" />
 				</include>
 				<include content="InfoDialogButton">
-					<param name="id" value="5010" />
+					<param name="id" value="8" />
```

Comment:
There's already button id ```8``` on ln 83. Better remove this one and make that one visible on ```Window.IsActive(PVRGuideInfo) | Window.IsActive(PVRRecordingInfo)```